### PR TITLE
Card starts evaluation UI

### PR DESCRIPTION
**Pull request by Joanartdacosta**
add a new component that adds starts according evaluation

### CODE REVIEW

File: app/components/common/UI/cards/card_stars/StarsEvaluation.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import StartWhite from "../../../icons/stars/StarWhite";
```
Comment (doublethink13) [inline app/components/common/UI/cards/card_stars/StarsEvaluation.js:1]: @

File: app/components/common/UI/cards/card_stars/StarsEvaluation.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import StartYellow from "../../../icons/stars/StarYellow";
+
+export default function StartsEvaluation(props) {
+  console.log(props.evaluation - 5);
```
Comment (doublethink13) [inline app/components/common/UI/cards/card_stars/StarsEvaluation.js:5]: console log

File: app/components/common/UI/cards/card_stars/StarsEvaluation.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/components/common/UI/cards/card_stars/StarsEvaluation.js]: done

File: app/components/common/UI/cards/card_stars/StarsEvaluation.js (hunk 2 above)
Comment (Joanartdacosta) [reply app/components/common/UI/cards/card_stars/StarsEvaluation.js]: deleted

**Pull request merged**